images/debian/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RELEASE
+
+FROM debian:${RELEASE}
+
+# udev is needed for booting a "real" VM, setting up the ttyS0 console properly
+# kmod is needed for modprobing modules
+# systemd is needed for running as PID 1 as /sbin/init
+# Also, other utilities are installed
+RUN apt-get update && apt-get install -y \
+      curl \
+      dbus \
+      kmod \
+      iproute2 \
+      iputils-ping \
+      net-tools \
+      openssh-server \
+      sudo \
+      systemd \
+      udev \
+      vim-tiny \
+      wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+# cleanup machine-id
+    echo "" > /etc/machine-id && echo "" > /var/lib/dbus/machine-id && \
+# inform systemd it is running containerized
+    mkdir -p /run/systemd && echo docker > /run/systemd/container && \
+# This container image doesn't have locales installed. Disable forwarding the
+# user locale env variables or we get warnings such as:
+# bash: warning: setlocale: LC_ALL: cannot change locale
+    sed -i -e 's/^AcceptEnv LANG LC_\*$/#AcceptEnv LANG LC_*/' /etc/ssh/sshd_config && \
+# remove ssh autogenerated ssh_host-* keys
+#   rm -f /etc/ssh/ssh_host* && \
```

Comment:
any practical use-case why you'd do this? security reasons?

Reply:
It makes sense not to bake keys into a published base image :+1:

The user should be expected to perform their own key-management unless we publish weak keys and have a standard runtime key generator like Vagrant does.